devfile.yaml

```diff
@@ -0,0 +1,122 @@
+apiVersion: 1.0.0
+metadata:
+  name: che-plugin-broker
+projects:
+  - name: che-plugin-broker
+    source:
+      location: 'https://github.com/eclipse/che-plugin-broker.git'
+      type: git
+      branch: master
+    clonePath: src/github.com/eclipse/che-plugin-broker
+components:
+  - id: ms-vscode/go/0.11.0
+    type: chePlugin
+    alias: ms-vscode.go
+
+  - type: dockerimage
+    image: 'eclipse/che-plugin-broker-dev:next'
+    alias: dev
+    mountSources: true
+    memoryLimit: 1G
+    env:
+      - value: '/go:$(CHE_PROJECTS_ROOT)'
+        name: GOPATH
+      - value: /tmp/.cache
+        name: GOCACHE
+    volumes:
+      - name: plugins
+        containerPath: /plugins
+
+  - type: dockerimage
+    image: 'quay.io/eclipse/che-plugin-registry:nightly'
+    alias: plugin-registry
+    endpoints:
+      - name: plugin-registry
+        attributes:
+          protocol: http
+          public: 'true'
+          discoverable: 'true'
+          type: plugin-registry
+          secure: 'false'
+        port: 8080
+    memoryLimit: 256M
+
+commands:
+  - name: compile "init" plugin broker
+    actions:
+      - workdir: /projects/src/github.com/eclipse/che-plugin-broker/brokers/init/cmd
+        type: exec
+        command: >-
+          printf 'Start "init" plugin broker compilation...\n' && 
+          CGO_ENABLED=0 GOOS=linux go build -a -ldflags '-w -s' -installsuffix cgo -o init-plugin-broker main.go &&
```

Comment:
Consider using the `Makefile` included -- this could be `make build-init` with workdir set to `/projects/src/github.com/eclipse/che-plugin-broker/`. This creates the binary in `/projects/src/github.com/eclipse/che-plugin-broker/` so if you go this route, the run commands will also need to be updated.